### DISCUSSION TITLE
Add PII Redaction

### DISCUSSION THREAD

**Discussion by patescher** [deepgram/community#251] [General help] [answered]
Hi, are you currently developing, or do you have an ETA, for enhanced Privacy Redaction, such that PIIs can be redacted? eg name, DOB, address, phone number, etc

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @patescher, thanks for the inquiry. Deepgram offers [redaction](https://developers.deepgram.com/docs/redaction) for English, with the options of credit card information, social security numbers, and aggressive redaction of all numbers. 

We do offer custom PII redaction options. If you're interested in discussing them, please reach out to me.
